README.md

```diff
@@ -1,38 +1,56 @@
 # Space Menace
+
 An action 2D platformer made with the [Amethyst](https://amethyst.rs/) game engine.
 
-This is my *firstish* attempt at a game (I did write a basic snake game some time back). Through this project, I aim to get better at coding in [Rust](https://www.rust-lang.org/), get familiar with the Amethyst game engine and start learning game development in general. I also hope that this project will help other Amethyst and game dev beginners like me in some way or the other.
+This is my _firstish_ attempt at a game (I did write a basic snake game some time back). Through this project, I aim to get better at coding in [Rust](https://www.rust-lang.org/), get familiar with the Amethyst game engine and start learning game development in general. I also hope that this project will help other Amethyst and game dev beginners like me in some way or the other.
 
-Currently, it is a :warning:**WORK IN PROGRESS**:warning: and there is still a lot to be done before it reaches a playable state. Also, the code is far from perfect and there is a lot of scope for improvement. I will keep enhancing the code incrementally, as I go along. 
+Currently, it is a :warning:**WORK IN PROGRESS**:warning: and there is still a lot to be done before it reaches a playable state. Also, the code is far from perfect and there is a lot of scope for improvement. I will keep enhancing the code incrementally, as I go along.
 
-## Running the game
-**Note:** This game requires Rust nightly.
+## Configure Cargo.toml
+
+The default render target feature is set to `metal` for macOS users.
+
+```toml
+# Cargo.toml
+
+[features]
+default = ["metal"]
+metal = ["amethyst/metal"]
+vulkan = ["amethyst/vulkan"]
 ```
+
+If you are on Windows or Linux, you will have to set that default to `vulkan`:
+
+```toml
+# Cargo.toml
+
+[features]
+default = ["vulkan"]
+...
```

Comment:
How is the reddish background color showing up for the `...`? Is this intentional?

Reply:
Sorry, I edited the README in VS Code and I forgot to comment it out, as it doesn't get highlighted there. I'll fix that